css/reset-css.md

```diff
@@ -2,64 +2,37 @@
 
 ### Projected Time
 
-Example: 30-45 minutes
-- Lesson: ? min
-- Guided Practice: ?-? min
-- Independent Practice: ? min
-- Check for Understanding: ?-? min
+About 30 minutes
+- Lesson: 4.5 min
+- Guided Practice: 10 min
+- Check for Understanding: 15 min
 
 ### Prerequisites
+- [HTML Lesson](https://github.com/Techtonica/curriculum/blob/master/html/html.md)
 - [CSS Lesson](/css/css.md)
 
 ### Motivation
-
-Here's why this topic should be learned. Strengthen this section by including a company that uses this skillset, and a screenshot example if possible.
+- In order to deal with the inconsistencies between user agent stylesheets across different browsers, CSS resets were born.
+- A CSS reset is a set of styles you load prior to your other styles, to remove browser built-in styles. 
 
 ### Objectives
 
 **Participants will be able to:**
-
-- leave this lesson knowing this
-- and how to do this on their own
-- Even more things
-- Even more things
+- Understand what is reset.css & why it is used.
+- Understand browser styling differences.
 
 ### Specific Things To Teach
 
-- about reset.css
+- how to use reset.css
+
 
 ### Materials
 
-- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
-- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
+- [Default browser styles and CSS Resets article (4 min read)](https://medium.com/@tinydinosaur/a-wordy-history-of-default-browser-styles-and-css-resets-befdd614d93b)
+- [Default browser styles and CSS Resets video (4.5 min watch)](https://www.youtube.com/watch?v=xLP8IM3jkgE)
 
 ### Lesson
-
-Here's text about introducing something and how it works.
-
-- [Be sure to link the slideshow, video,](google.com)
-- [or other materials you expect them to follow as part of the lesson.](google.com)
-
-Build on the first information. Have apprentices guess things, do an activity, etc.
-
-Make sure to mention these things:
-
-- Things
-    - This is a sub-thing
-- More things
-- Even more things
-- Even more things
-
-
-### Common Mistakes / Misconceptions
-
```

Comment:
You could add some common mistakes/misconception that newbies make.

Reply:
ok

Reply:
😃